gravitee-am-reporter/gravitee-am-reporter-kafka/src/main/java/io/gravitee/am/reporter/kafka/KafkaReporterConfiguration.java

```diff
@@ -25,43 +27,15 @@
  * @author Florent Amaridon
  * @author Visiativ
  */
+@Setter
+@Getter
 @JsonInclude(JsonInclude.Include.NON_ABSENT)
 public class KafkaReporterConfiguration implements ReporterConfiguration {
 
   private String bootstrapServers;
   private String topic;
   private String acks;
+  private String schemaRegistryUrl;
```

Comment:
**question**: how do we manage authentication on this registry ?

Reply:
To add authentication we can use additionalProperties. For example to use basic auth we have to set `basic.auth.credentials.source=USER_INFO`
`basic.auth.user.info=username:password`
Using additional Properties ssl certificate can be set as well.